pkg/resource/feature_group/testdata/feature_group/cr/fg_cmd_create_initiated.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: FeatureGroup
+metadata:
+  creationTimestamp: null
+  name: unit-testing-feature-group
+spec:
+  eventTimeFeatureName: EventTime
+  featureDefinitions:
+  - featureName: TransactionID
+    featureType: Integral
+  - featureName: EventTime
+    featureType: Fractional
+  featureGroupName: unit-testing-feature-group
+  offlineStoreConfig:
+    s3StorageConfig:
+      s3URI: s3://source-data-bucket-592697580195-us-west-2/sagemaker/feature-group-data
+  recordIdentifierFeatureName: TransactionID
+  roleARN: arn:aws:iam::123456789012:role/ack-sagemaker-execution-role
+status:
+  ackResourceMetadata:
+    arn: arn:aws:sagemaker:us-west-2:123456789012:feature-group/unit-testing-feature-group
+    ownerAccountID: ""
+  conditions: []
```

Comment:
shouldn't this have a featureGroupStatus `Creating` as well?

Reply:
When Creating a feature group, it first calls create, which verifies all the input fields are of an acceptable format, as well as assigns a resource arn. After this, a ReadOne is called, where the status, conditions, and any additional fields (added through the custom_delta) are added.